### ISSUE TITLE
Suffix icon & Rounded corners

### ISSUE THREAD

**Issue by Alyaqdhans** [GaspardMerten/date_field#84]
![image](https://github.com/user-attachments/assets/3eeb7e29-2329-4439-bfa1-30898a333d47)

I can't round the corners of the date field, add `borderRadius` property like the Dropdown widget
![image](https://github.com/user-attachments/assets/b15fcccd-d3f3-424f-81ca-a1a59f5f1a0d)

make the suffix icon position similar to the dropdown arrow (a bit left), and if possible add suffix icon color property
![image](https://github.com/user-attachments/assets/01ff8103-da4d-4037-a28b-5d04267ba18d)

**Comment by torbenkeller** [comment]
Can you provide some code to reproduce this?

**Comment by Alyaqdhans** [comment]
just add this property
```
decoration: InputDecoration(
  border: OutlineInputBorder(borderRadius: BorderRadius.circular(15)),
),
```

**Comment by GaspardMerten** [comment]
Hi @Alyaqdhans, the border radius behaviour is now fixed in Flutter v5.3.3.
